### PR TITLE
Add function check_if_game_finished.

### PR DESCRIPTION
**Pull request by boryskova**
Add a file with the function check_if_game_finished.

### CODE REVIEW

File: src/check_if_game_finished.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""Function to check if the game is finished."""
+from frozendict import frozendict
+
+
```
Comment (Ipceeh) [inline src/check_if_game_finished.py:4]: No need in `empty_hole_char`

File: src/check_if_game_finished.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                  }
+
+    if move['move_type'] == 'safe' and move['coordinates'] in bombs:
+        game_status['status'] = 'end_game'
+        game_status['end_game_status'] = 'lose'
+
+    if move['move_type'] == 'safe' and move['coordinates'] not in bombs:
+        open_safe_cells = 0
+        for row in matrix:
+            open_safe_cells += row.count(empty_hole_char)
```
Comment (Ipceeh) [inline src/check_if_game_finished.py]: Wrong data type for matrix

File: src/check_if_game_finished.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+        if open_safe_cells == (len(matrix) ** 2 - len(bombs)):
```
Comment (Ipceeh) [inline src/check_if_game_finished.py]: len(matrix) * len(matrix[0])

**Review by Ipceeh — CHANGES_REQUESTED** [review]

**Review by Ipceeh — APPROVED** [review]
Looks good for me!

**Review by oksanastep — APPROVED** [review]

**Pull request merged**